plantuml-parser-cli/src/main/java/com/shuzijun/plantumlparser/cli/options/showdefaultconstructors/ShowDefaultConstructorsOption.java

```diff
@@ -0,0 +1,35 @@
+package com.shuzijun.plantumlparser.cli.options.showdefaultconstructors;
+
+import com.shuzijun.plantumlparser.cli.options.CliOption;
+import com.shuzijun.plantumlparser.core.ParserConfig;
+import java.util.Arrays;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+
+public class ShowDefaultConstructorsOption implements CliOption {
+
+    protected final String optName = "sdfctr";
```

Comment:
I personally like "sdctr" better than "sdfctr" (since it is more similar to sctr), but it is up to you whether to change it.